examples/chatpp.cc

```diff
@@ -99,7 +99,7 @@ static void parse_command(meshlink::mesh *mesh, char *buf) {
 			if(!nodes) {
 				fprintf(stderr, "Could not get list of nodes: %s\n", meshlink::strerror());
 			} else {
-				printf("%zu known nodes:", nnodes);
+				printf("%lu known nodes:", nnodes);
```

Comment:
This is wrong on some platforms (I think on Windows `long` is always 32 bits, while `size_t` can be 64 bits). If `%zu` is not available, then you can use `%lu`, but then make sure to cast the argument to `unsigned long`, like:
```
printf("%lu known nodes:", (unsigned long)nnodes);
```